src/lib/config/config.ts

```diff
@@ -1,25 +1,15 @@
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-nocheck
-import {
-	KINDE_AUDIENCE,
-	KINDE_CLIENT_ID,
-	KINDE_CLIENT_SECRET,
-	KINDE_ISSUER_URL,
-	KINDE_POST_LOGIN_REDIRECT_URL,
-	KINDE_POST_LOGOUT_REDIRECT_URL,
-	KINDE_REDIRECT_URL,
-	KINDE_SCOPE,
-	KINDE_AUTH_WITH_PKCE
-} from '$env/static/private';
+import {env} from '$env/dynamic/private';
 
 export const kindeConfiguration = {
-	authDomain: KINDE_ISSUER_URL,
-	clientId: KINDE_CLIENT_ID,
-	logoutRedirectURL: KINDE_POST_LOGOUT_REDIRECT_URL,
-	redirectURL: KINDE_REDIRECT_URL,
-	audience: KINDE_AUDIENCE,
-	scope: KINDE_SCOPE,
-	clientSecret: KINDE_CLIENT_SECRET,
-	loginRedirectURL: KINDE_POST_LOGIN_REDIRECT_URL,
-	authUsePKCE: [true, 'true'].includes(KINDE_AUTH_WITH_PKCE)
+	authDomain: env.KINDE_ISSUER_URL,
+	clientId: env.KINDE_CLIENT_ID,
+	logoutRedirectURL: env.KINDE_POST_LOGOUT_REDIRECT_URL,
+	redirectURL: env.KINDE_REDIRECT_URL,
+	audience: env.KINDE_AUDIENCE,
+	scope: env.KINDE_SCOPE,
+	clientSecret: env.KINDE_CLIENT_SECRET,
+	loginRedirectURL: env.KINDE_POST_LOGIN_REDIRECT_URL,
+	authUsePKCE: [true, 'true'].includes(env.KINDE_AUTH_WITH_PKCE)
```

Comment:
Small nitpick this, but I don't think you need explicitly check for `true` here. I think `authUsePKCE: env.KINDE_AUTH_WITH_PKCE === 'true'` would perhaps make more sense ?

Reply:
Thanks for this feedback @atifcppprogrammer.
I will pass this feedback on.